Clinical-Genomics/BALSAMIC
# Housekeeper and scout templates

This PR adds housekeeper and scout support:

1. creates a housekeeper yaml file with files to preserve.
2. creates a crude template for scout yaml file.

How to test:
- all tests pass
- run on a completed analysis, and it should create a `delivery_report` directory

Expected outcome:
- *.hk and *.scout.yaml should be created.
